iOSJenkinsCI/ViewController.swift

```diff
@@ -16,7 +16,7 @@ class ViewController: UIViewController {
         let button = UIButton(frame: CGRect(x: 50, y: 50, width: 100, height: 20))
         self.view.addSubview(button)
         button.rx.tap.subscribe { _ in
-            debugPrint("fjsldjfskdf")
+            debugPrint("fjsldjfskdf")  
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/corey-lin/iOSJenkinsCI/blob/37b404cb7f6f0dcf1b34fe4f5a4109c679f28943/iOSJenkinsCI/ViewController.swift#L19"></span>Lines should not have trailing whitespace.</td>
    </tr>
  </tbody>
</table>